api.go

```diff
@@ -1849,3 +1849,67 @@ func ctxSetSession(r *http.Request, s *SessionState) {
 	}
 	context.Set(r, SessionData, s)
 }
+
+func ctxGetAuthToken(r *http.Request) string {
+	if v := context.Get(r, AuthHeaderValue); v != nil {
+		return v.(string)
+	}
+	return ""
+}
+
+func ctxSetAuthToken(r *http.Request, t string) {
+	if t == "" {
+		panic("setting a nil context AuthHeaderValue")
+	}
+	context.Set(r, AuthHeaderValue, t)
+}
+
+func ctxGetTrackedPath(r *http.Request) string {
+	if v := context.Get(r, TrackThisEndpoint); v != nil {
+		return v.(string)
+	}
+	return ""
+}
+
+func ctxSetTrackedPath(r *http.Request, p string) {
+	if p == "" {
+		panic("setting a nil context TrackThisEndpoint")
+	}
+	context.Set(r, TrackThisEndpoint, p)
+}
+
+func ctxGetDoNotTrack(r *http.Request) bool {
```

Comment:
I prefer `Get` for consistency. Also "is do not track" seems like the wrong sentence, since the argument is a request. Perhaps "has do not track". But I still prefer get for consistency with the rest of the funcs like it.